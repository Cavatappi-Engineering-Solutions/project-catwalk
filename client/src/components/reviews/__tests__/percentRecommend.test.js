/* eslint-disable no-undef */
import React from 'react'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import PercentRecommend from '../percentRecommend.jsx'

it('renders correctly with recommendations', () => {
  const { getByText } = render(<PercentRecommend recommended={{ true: '4', false: '1' }} />)
  getByText('80% of reviews recommend this product')
})

it('renders correctly without recommendations', () => {
  const tree = renderer.create(<PercentRecommend recommended={{ false: '3' }} />).toJSON()
  expect(tree).toMatchSnapshot()
})
