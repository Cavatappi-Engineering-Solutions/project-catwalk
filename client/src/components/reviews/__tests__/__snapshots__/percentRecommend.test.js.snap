// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without recommendations 1`] = `
<div
  id="percent-recommend"
/>
`;
